frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx

```diff
@@ -225,6 +225,25 @@ export const CreateVM: React.FC<RouteComponentProps> = ({ location }) => {
     }
   }, [loaded, initData, templates, userTemplates, selectedTemplate, t]);
 
+  React.useMemo(() => {
+    const temp = selectedTemplate?.variants?.[0];
+    const requestedPvcSize =
+      temp?.objects?.[0]?.spec?.dataVolumeTemplates[0]?.spec?.pvc?.resources?.requests?.storage;
+
+    if (requestedPvcSize) {
+      const [value] = requestedPvcSize.match(/\d+/g);
+      const [unit] = requestedPvcSize.match(/[a-zA-Z]+/g);
```

Comment:
@rawagner also here, do we have a better option?

Reply:
maybe `stringValueUnitSplit` can help here ?

Reply:
`stringValueUnitSplit` or use `getSize()` from `DataVolumeWrapper`